### PR TITLE
Update Scenario1_BasicFunctionality.xaml.cs

### PR DESCRIPTION
**Pull request by hyunju-park**
add trigger event in the barcode scanner project.

### CODE REVIEW

**Comment by hyunju-park** [comment]
Add trigger event

**Comment by hyunju-park** [comment]
Add trigger event

**Comment by hyunju-park** [comment]
Add trigger event

File: Samples/BarcodeScanner/cs/Scenario1_BasicFunctionality.xaml.cs [hunk 1]
```diff
@@ -136,6 +136,10 @@ protected override void OnNavigatedFrom(NavigationEventArgs e)
 
                     // after successfully claiming, attach the datareceived event handler.
                     claimedScanner.DataReceived += claimedScanner_DataReceived;
```
Comment (piojugi) [inline Samples/BarcodeScanner/cs/Scenario1_BasicFunctionality.xaml.cs]: > #

**Comment by oldnewthing** [comment]
Adding an event handler that does nothing doesn't really demonstrate anything.

**Pull request closed without merging**